hedera-node/src/test/java/com/hedera/services/ledger/HederaLedgerTest.java

```diff
@@ -78,6 +81,48 @@ private void setup() {
 		setupWithMockLedger();
 	}
 
+	@Test
+	void checkAccountUsabilityWorks() {
+		final long balance = 0;
+		final long expiry = 123;
+		final AccountID testId = AccountID.newBuilder().setAccountNum(123L).build();
+		given(accountsLedger.exists(testId)).willReturn(false);
+
+		assertEquals(INVALID_ACCOUNT_ID,  subject.usableOrElse(testId, INVALID_ACCOUNT_ID));
+
+		given(accountsLedger.exists(testId)).willReturn(true);
+		given(accountsLedger.get(testId, IS_DELETED)).willReturn(true);
+
+		assertEquals(ACCOUNT_DELETED,  subject.usableOrElse(testId, ACCOUNT_DELETED));
+
+		validator = mock(OptionValidator.class);
+//		given(validator.isAfterConsensusSecond(expiry)).willReturn(false);
+//		given(accountsLedger.exists(testId)).willReturn(true);
+//		given(accountsLedger.get(testId, IS_DELETED)).willReturn(false);
+//		given(accountsLedger.get(testId, IS_SMART_CONTRACT)).willReturn(false);
+//		given(accountsLedger.get(testId, BALANCE)).willReturn(balance);
+//		given(accountsLedger.get(testId, EXPIRY)).willReturn(expiry);
+//
+//		assertEquals(ACCOUNT_EXPIRED_AND_PENDING_REMOVAL,  subject.usableOrElse(testId, ACCOUNT_DELETED));
```

Comment:
same here

Reply:
This needs fixing. Will uncomment and fix